elide-async/src/test/resources/bb31ca4e-ed8f-4be0-a0f3-12099fb9263f

```diff
@@ -0,0 +1 @@
+test
```

Comment:
Maybe we can move this under a subdirectory with a descriptive name so it is clear what this file is about.